apps/stripe/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  plugins: [],
+  test: {
+    include: ["src/**/*.test.{ts,tsx}"],
+    passWithNoTests: true, // TODO: remove this after we add first test
+    setupFiles: "./__tests__/setup.ts",
+    css: false,
+    alias: {
+      "@/": new URL("./src/", import.meta.url).pathname,
+      "@/generated": new URL("./generated/", import.meta.url).pathname,
+      "@/package.json": new URL("./package.json", import.meta.url).pathname,
```

Comment:
I think it would be easier if we use aliases more, so maybe let's keep that in mind? 
I would even consider eslint rule to enforce alias imports